### PR TITLE
Add Unicorn chain

### PR DESCRIPTION
**Pull request by risasoft**

### CODE REVIEW

File: cosmos/unicorn-420.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "chainId": "unicorn-420",
+  "chainName": "Unicorn",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/unicorn/chain.png",
+  "rpc": "https://rpc.unicorn.meme",
+  "rest": "https://rest.unicorn.meme",
+  "nodeProvider": {
+    "name": "Unicorn",
+    "website": "https://x.com/unicornandmemes"
+  },
```
Comment (editaahn) [inline cosmos/unicorn-420.json:10]: Please add email information

**Review by editaahn — COMMENTED** [review]
Hey, please make sure the JSON format is valid.

**Comment by steveklebanoff** [comment]
@editaahn This PR has an updated JSON can you take a look ? https://github.com/chainapsis/keplr-chain-registry/pull/808

**Comment by risasoft** [comment]
@editaahn edits made, please review

**Comment by editaahn** [comment]
merged at https://github.com/chainapsis/keplr-chain-registry/pull/808

**Comment by steveklebanoff** [comment]
Thanks for this!

When trying to mobile connect, I still see this though.  Does the mobile app need to be updated with the new metadata?

CleanShot 2024-09-20 at 17 32 10@2x

**Pull request closed without merging**